bucket/apifox.json

```diff
@@ -28,6 +28,7 @@
             "--user-data-dir=\"$dir\\UserData\""
         ]
     ],
+    "pre_uninstall": "Stop-Process -Name 'ApifoxAppAgent' -ErrorAction SilentlyContinue; Start-Sleep -Seconds 3;",

```

Comment:
Use `-Wait`-like method instead `Start-Sleep`.

Reply:
No usage of stop-process with -wait was retrieved. @HUMORCE 
https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.management/stop-process?view=powershell-7.3

Reply:
start-process tskill -args 'app.exe' -wait

Reply:
done, review pls.